Resources/public/js/views/ez-contenteditformview.js

```diff
@@ -56,20 +52,25 @@ YUI.add('ez-contenteditformview', function (Y) {
                 contentType = this.get('contentType'),
                 fieldDefinitions = contentType.get('fieldDefinitions'),
                 views = [],
-                that = this;
+                that = this,
+                config = this.get('config');
 
             Y.Object.each(fieldDefinitions, function (def) {
-                var EditView, view;
+                var EditView, view, fieldConfig;
 
                 try {
                     EditView = Y.eZ.FieldEditView.getFieldEditView(def.fieldType);
 
+                    if (config && config.editViews&& config.editViews[def.fieldType]) {
```

Comment:
Nitpick: missing space before `&&` (and of course s/editViews/fieldEditViews/ as mentioned in https://github.com/ezsystems/PlatformUIBundle/pull/177/files#r25071200)